Azure/azure-cli
# {Core} Add synapse endpoint and suffix.

**Description**  
This PR adds the synapse endpoint suffix for AzureCloud and AzureChinaCloud. Prepare for adding cli synapse module.

**Testing Guide**  

`az cloud show -n AzureCloud`
`az cloud show -n AzureChinaCloud`


---

This checklist is used to make sure that common guidelines for a pull request are followed.
